examples/example-app-v12-monorepo/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "example-app-v12-monorepo",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "jest -c=projects/app1/jest.config.js",
+    "test-esm": "NODE_OPTIONS=--experimental-vm-modules jest -c=projects/app1/jest-esm.config.js"
```

Comment:
sadly Windows is not happy with `NODE_OPTIONS`, this option seems to work only with Linux/Mac